tests/queries/0_stateless/02503_cache_on_write_with_small_segment_size.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CLICKHOUSE_CLIENT_SERVER_LOGS_LEVEL=none
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+function random {
+     cat /dev/urandom | LC_ALL=C tr -dc 'a-zA-Z' | fold -w ${1:-8} | head -n 1
+}
+
+${CLICKHOUSE_CLIENT} --multiline --multiquery -q "
+drop table if exists ttt;
+create table ttt (id Int32, value String) engine=MergeTree() order by tuple()  settings storage_policy='s3_cache_small_segment_size';
+insert into ttt select number, toString(number) from numbers(100000) settings throw_on_error_from_cache_on_write_operations = 1;
```

Comment:
Is it possible to reproduce some write error in tests to check different behavior of `throw_on_error_from_cache_on_write_operations` ?

Reply:
Yes, will do.

Reply:
After a second thought - I cannot make up how to do this - could be done with getting our of disk space, but for this case there are already checks and it will not throw.